service/dependencies.gradle

```diff
@@ -44,8 +47,8 @@ dependencies {
   implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: "1.2.31-SNAPSHOT"
 
   // Terra Landing Zone Service
-  implementation ('bio.terra:terra-landing-zone-service:0.0.367-SNAPSHOT')
-  implementation ('bio.terra:landing-zone-service-client:0.0.367-SNAPSHOT')
+  implementation ('bio.terra:terra-landing-zone-service:0.0.390-SNAPSHOT')
+  implementation ('bio.terra:landing-zone-service-client:0.0.390-SNAPSHOT')
```

Comment:
this is the version of LZS which upgraded its TCL; see https://github.com/DataBiosphere/terra-landing-zone-service/pull/546